.github/workflows/build-latex.yml

```diff
@@ -0,0 +1,38 @@
+name: Build LaTeX document
+on: [push]
+jobs:
+  build_latex:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Git repository
+        uses: actions/checkout@v1
```

Comment:
@Sammers21 I'd use `checkout@v2` here, new version has a lot of improvements, see https://github.com/actions/checkout#whats-new